### PR TITLE
Add incognito browser SEO page (Fixes #6712)

### LINKED ISSUE #6712: Launch new SEO page "Incognito browser"
| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | 2 |
| ENG | ? |
| SEO | ? |
| ANA | |
| DES | |
| COPY | |

Description: using the template established for SEO pages from the NA Desktop team, create the new page 'Incognito browser'

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Page is live before end of January 2019

#### Tasks
- [ ]  Assign the URL /technology/incognito-browser/
- [ ] Use the copy found in [this doc](https://docs.google.com/document/d/1cyHpu6mB2_-Dy1j9Gazw4OqTGymQ6gQTxL-Ys75rokE/edit) (includes meta data at the bottom)

Native Sketch file can be found here (in Abstract): https://share.goabstract.com/75a5ee3e-51e2-415f-8950-3ae3e70200ef

Look for the `Incognito Landing Page" layer.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds a new page at `/technology/incognito-browser/`

## Issue / Bugzilla link
#6712

## Testing
Demo: https://bedrock-demo-agibson.oregon-b.moz.works/en-US/technology/incognito-browser/

- [ ] Check for copy pasta errors? 🍝

### CODE REVIEW

**Comment by ejregithub** [comment]
I have this in front of stakeholders for QA signoff

File: bedrock/mozorg/templates/mozorg/incognito-browser.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    <h2 class="c-section-title-large">{{ _('Why go private/incognito?') }}</h2>
+
+    <p>
+    {% trans %}
+      Just because you’re using private browsing mode doesn’t mean you’re up to something
+      nefarious. Perhaps you want to keep your work and personal life separate. You might
+      share a computer or device and you don’t want your siblings snooping. You could be
+      shopping for a gift and you don’t want anything to spoil the surprise. Or maybe you
+      just want to limit the amount of data companies collect about you and you value privacy.
+      Incognito or private browsing mode is mode for any of these scenarios.
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/incognito-browser.html]: Typo: repeated "mode" but I presume the second one should be "made"

File: bedrock/mozorg/templates/mozorg/incognito-browser.html [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    <p>
+    {% trans %}
+      Cookies were first used to customize websites, keep track of shopping carts, and maintain
+      online account security, but today most are used to help companies serve targeted ads.
+    {% endtrans %}
+    </p>
+
+    <p>
+    {% trans %}
+      Here’s how it works: You visit a site, an advertiser leaves a  cookie on your browser.
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/incognito-browser.html]: Nit: extra space between `a  cookie`. Not a big deal but could make l10n troublesome (no idea if there are any plans to localize this page anyway).

File: bedrock/mozorg/templates/mozorg/incognito-browser.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+      <a href="{{ tor }}">Tor Browser</a> can truly mask your online activity. It bounces traffic
+      through multiple servers around the globe, making it difficult to track that traffic. The
+      website you visit really has no idea where you are, only the approximate location of the last
+      server your request was routed through. But again, even Tor proxy won’t stop third-party
+      advertisers from installing cookies in your browser. Tor Browser deletes all cookies when
+      closed. People can also start a new session in Tor Browser to clear them as well.
+    {% endtrans %}
+    </p>
+
+    <h3 class="c-section-title">{{ _('Incognito: Tldr') }}</h3>
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/incognito-browser.html]: Nit: I think this is typically written as "tl;dr", with a semicolon, and for title case here it should even be "TL;DR" in all caps. I question using such meme culture jargon in this kind of copy at all, but that's not my call. But if we're going to do it we should get it right.

Also if there is any plan to localize this text we should add an explanatory 1l0n comment, maybe with an alternative for translation. Maybe "Incognito: The Short Version"?

**Comment by alexgibson** [comment]
Thanks @craigcook - rebased and updated

**Comment by alexgibson** [comment]
Added another small commit which adds a secondary CTA at the bottom of the page (I missed this in the original mockup)

**Pull request merged**